docs/index.rst

```diff
@@ -56,7 +56,12 @@ variety of platforms.
 
     cloud/emr
 
+.. toctree::
+    :caption: Demo Notebooks 
+    :maxdepth: 2
 
+    notebooks/mango-python-alignment-executed.ipynb
```

Comment:
How are these notebooks generated?

Reply:
By me manually running them after starting 
```
bin/mango-notebook
```
and "save-as" them with new file name in the docs/notebooks/

Also, I had to add a "Title" in first cell and change that cell to "Markdown" type for nbphinx to recognize it and build a link in the html.
  
It would prefer to auto run these notebooks from the single original notebook during the sphinx build, but right now the needed Spark context to not available, and would require adding those imports, which would make it diverge from the original notebook meant to be run via mango-notebook.

I'll look into that further if you want me to.

Reply:
Can you also add a Markdown title to the original notebook?

Automating the process would be the way to go. What imports would we need? Do you have an idea of how to do this? You could programmatically execute the notebook using nbconvert: https://nbconvert.readthedocs.io/en/latest/execute_api.html